study.md

```diff
@@ -44,21 +59,52 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+
+With the MVC model, "view" displays the view model and sends user input to the view model.  In the JS MVVM framework, view is markup.  This helps to separate the data from the presentation of the data, with "view" being the layer that reads data from the models.  "View state" is a repository in an ASP.NET page that can store values that need to be retained during postback
```

Comment:
Most of this is right but disregard the ASP.NET stuff, that doesn't necessarily apply to what we are talking about when we say view states.